packages/interbit-ui-components/src/assets/icons/apps.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"/><path d="M0 0h24v24H0z" fill="none"/></svg>
```

Comment:
@karikushi I'm sorry, what did you mean? This svg is an apps icon from https://material.io/tools/icons/?icon=apps&style=baseline (url from spec), but in styleguidist examples I use AppsIcon from @material-ui/icons.

Reply:
On second look, it seems like you don't need this .svg asset at all as you're using the AppsIcon component. If it can be removed, let's get remove it.